src/documentToNodes.js

```diff
@@ -1,18 +1,35 @@
-import * as R from 'ramda'
-import * as RA from 'ramda-adjunct'
+import {
+  compose,
+  cond,
+  test,
+  always,
+  T,
+  identity,
+  prop,
+  find,
+  propEq,
+  pick,
+  then,
+  fromPairs,
+  map,
+  toPairs,
+  omit,
+  propOr,
+} from 'ramda'
+import { allP, mapIndexed } from 'ramda-adjunct'
 import pascalcase from 'pascalcase'
 
 const IMAGE_FIELD_KEYS = ['dimensions', 'alt', 'copyright', 'url', 'localFile']
```

Comment:
Maybe this should be in `constants.js` since it's a constant

Reply:
Moved over.